src/query/config/src/config.rs

```diff
@@ -1364,6 +1364,9 @@ pub struct QueryConfig {
     #[clap(long, default_value = "localhost")]
     pub rpc_tls_query_service_domain_name: String,
 
+    #[clap(long, default_value = "0")]
+    pub rpc_client_timeout_secs: u64,
```

Comment:
Give a reasable default value?

Reply:
this default value 0 is intended to keep the same behaviour as before, because I'm a bit afraid of setting a timeout here may expose some additional errors on the communication between the instances, I would try a good configuration value in my environment, and put it default in another PR